### PR TITLE
Movies page,Pagination

### PR DESCRIPTION
**Pull request by Dawid-Zieba**
I've done movie details folder which contains movie detail's saga,slice,where it is not connected yet. Added Error Page and NoResult folder and pagination is now stylized and I'm working on it to make it work

### CODE REVIEW

**Review by MateuszPiaszczyk — APPROVED** [review]
Generally looks good :)

File: src/features/MoviesList/MovieDetails/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    const movieInfo = useSelector(selectMovieDetails);
+    const query = useQueryParameter("search")
+
+    useEffect(() => {
+        if (id) {
+            dispatch(fetchMovieDetails(id))
+        }
+    }, []);
+
+    console.log(movieInfo)
```
[the remark this answers is not in the document]
Comment (Dawid-Zieba) [reply src/features/MoviesList/MovieDetails/index.js:20]: I just wanted to check if it works, I forgot to remove it

File: src/features/MoviesList/MovieDetails/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    const status = useSelector(selectMovieDetailsStatus)
+    if (status ==="error") return <ErrorPage />
+    
+    return (
+        <>
+        
+        </>
```
[the remark this answers is not in the document]
Comment (Dawid-Zieba) [reply src/features/MoviesList/MovieDetails/index.js:27]: will be finished, for now this is just a rough code skeleton. I'm sorry :pray:

File: src/features/MoviesList/MovieDetails/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (MateuszPiaszczyk) [reply src/features/MoviesList/MovieDetails/index.js]: Good 👍

File: src/features/MoviesList/MovieDetails/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (MateuszPiaszczyk) [reply src/features/MoviesList/MovieDetails/index.js:26]: ok i understand 👍

**Pull request merged**